go.mod

```diff
@@ -48,9 +48,9 @@ require (
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
 	github.com/golang/mock v1.4.4
-	github.com/golang/protobuf v1.4.3
+	github.com/golang/protobuf v1.5.0
```

Comment:
uh oh... was this really required on the runc v1.0.0 bump? I need to check if this is in the set of problematic grpc/protobuf version crossings that https://github.com/kubernetes/kubernetes/pull/100488 was required to fix

Reply:
yeah... runc 1.0.0 bumped to google.golang.org/protobuf@v1.26.0 which requires this

Reply:
> the new version of proto libraries brought deprecations with them (see https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/103392/pull-kubernetes-verify/1410422073385291776)
> 
> see fix in master in [ff2c614](https://github.com/kubernetes/kubernetes/commit/ff2c61424598924a681e66d71e8deeb21b05101e)

Right, I forgot about this one.

I used to have the exactly same fix in 102508 which I removed once #100488 got merged.

Cherry-picked ff2c61424598924a681e66d71e8dee to this PR.